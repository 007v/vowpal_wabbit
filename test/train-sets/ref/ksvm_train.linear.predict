0
0.532215
0.049905
-0.405208
-0.434059
-0.587939
-0.372273
-0.555874
-0.544540
-0.630671
-0.630536
-0.563670
-0.661388
-0.745142
-0.666458
-0.328404
-0.264202
-0.212201
-0.304081
-0.368439
-0.527142
-0.530868
-0.410699
-0.209956
-0.381080
-0.312834
-0.539583
-0.816087
-0.198811
-0.348430
-0.499391
0.016406
-0.608167
-0.487777
-0.456441
-0.481600
-0.519893
-0.162202
-0.248895
0.068481
-0.387022
-0.265605
-0.364842
-0.227405
-0.523090
-0.582978
-0.145217
-0.356795
-0.674866
-0.384307
-0.442993
-0.251650
-0.423747
-0.476865
-0.330325
0.322735
-0.295108
-0.470508
-0.442613
-0.290616
0.007775
-0.014116
0.214935
-0.605579
-0.171550
-0.161115
-0.324490
-0.294927
-0.469393
-0.397332
-0.336532
-0.057239
-0.397650
-0.551215
0.256639
0.156320
-0.278219
-0.411680
-0.497529
-0.491182
-0.351296
-0.319970
0.126865
-0.230917
-0.589398
-0.480772
-0.349415
0.025086
-0.111599
-0.901352
-0.439024
-0.278569
0.651353
-0.271293
0.205488
-0.823277
-0.219241
-0.062077
-0.600187
-0.518833
-0.204766
-0.371806
-0.017113
-0.225954
0.035401
-0.079095
-0.575424
-0.435617
-0.190856
-0.067796
-0.051857
-0.241977
-0.130728
-0.421785
0.293078
-0.506490
-0.383161
0.119347
-0.288299
-0.511861
-0.379584
-0.013920
-0.110147
-0.412731
0.023198
-0.021917
-0.550337
0.108373
-0.721810
-0.045962
-0.190833
0.240799
-0.307808
-0.522218
-0.365551
-0.469095
-0.064000
0.224292
0.673978
-0.593011
-0.494779
-0.520469
0.782806
0.997811
-0.171213
0.274153
-0.116529
0.915562
-0.818817
-0.208733
0.122853
-0.015868
0.506190
-0.145014
-0.444999
0.422994
0.826604
-0.207110
0.893037
0.031733
0.844008
-0.716563
-0.209344
0.046088
-0.123056
0.074668
0.362799
-0.499410
-0.162714
0.556965
0.522456
-0.815757
-0.404072
0.405727
-0.500500
0.130765
-0.778147
0.892407
0.150522
0.297153
-0.164014
-0.158769
0.268740
-0.501942
-0.063745
-0.035363
0.358131
-0.356254
-0.296796
0.233656
0.642436
0.061063
-0.023925
-0.416059
-0.139825
0.211572
0.237502
-0.396101
-0.082283
0.079088
-0.213712
0.095503
0.058536
-0.536447
-0.436919
-0.364387
0.669956
-0.072996
-0.113464
-0.031879
-0.076666
-0.729919
-0.514381
-0.002164
1.128606
-0.732797
-0.518685
-0.273353
-0.126255
-0.425711
0.055404
-0.678571
-0.539010
-0.112445
0.217527
-0.396729
1.347297
0.567375
-0.350973
-0.765781
-0.592243
-0.512913
-0.650969
0.020591
0.303422
-0.555471
-0.543751
0.444679
1.008771
0.325962
0.441904
-0.312483
-0.961506
0.712582
-0.106917
0.412000
0.183662
-0.476889
-0.522137
0.538492
