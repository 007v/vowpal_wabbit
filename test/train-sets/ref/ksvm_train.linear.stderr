using l2 regularization = 1
predictions = ksvm_train.linear.predict
Lambda = 1
Kernel = linear
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.266108 1.532215            2            2.0  -1.0000   0.5322      103
1.044228 0.822348            4            4.0  -1.0000  -0.4052      134
0.925331 0.806433            8            8.0  -1.0000  -0.5559      145
0.867681 0.810032           16           16.0   1.0000  -0.3284       23
0.888633 0.909584           32           32.0  -1.0000   0.0164       31
0.887705 0.886778           64           64.0  -1.0000  -0.6056       60
0.881002 0.874300          128          128.0   1.0000   0.1084      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.780617
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 241
Number of kernel evaluations = 175337 Number of cache queries = 106071
Total loss = 195.154129
Done freeing model
Done freeing kernel params
Done with finish 
